### ISSUE TITLE
option --onlyinvalid does nothing

### ISSUE THREAD

**Issue by hugoledoux** [tudelft3d/val3dity#134]
but with the new report style, perhaps it makes no sense... the report reports on stats for the file, so having only the features having errors kind of break this. And breaks surely the web-report of @balazsdukai I reckon

so: delete that option?

**Comment by arruga** [comment]
Hugo,
I use the command line val3dity with the report in json, so I don't know what are the implications on the web-report.
I vote for keeping this option. For me it should be the standard behaviour and valid buildings should appear only in the global statistics. Otherwise, the report becomes too heavy and requires an authomatic treatment. 
It would be also very useful including the envelope of each problem building in the report.

Kind regards

**Comment by hugoledoux** [comment]
You have a good point indeed, will keep it but I need to clean and modify a few things for this. 

> It would be also very useful including the envelope of each problem building in the report.

I am not sure I get what you mean here

**Comment by arruga** [comment]
Asking for the envelop is an out of place request. I mean the bounding box of the building. In my case, the building id has no meaning and I have to search the id in the citygml file to find the building coordinates.

**Comment by balazsdukai** [comment]
I agree with @arruga , and we can adjust the web-report as necessary to fit the needs. Also, making --onlyinvalid default for reporting makes sense, as i can understand that whoever reads the report, does it mainly for getting info about the errors, not the valids.

**Comment by hugoledoux** [comment]
ok agreed that will fixed eventually.

for bbox: fair enough, I see your point. We can add this.

**Comment by hugoledoux** [comment]
I have decided to remove the option `--onlyinvalid` from val3dity, starting from v2.2

Reasoning:
1. it makes the report more confusing, in the past I had people actually asking if their building was validated (since I only outputted valid ones in v1)
2. the online report has been revamped and there's an easy filtering option to show only the invalid ones, see it at: http://geovalidation.bk.tudelft.nl/val3dity/browser/
3. the report doesn't get that big, does it?

aface5980e5621c2748f97ef6db0e444617fec3e
